pom.xml

```diff
@@ -16,8 +16,8 @@
     </parent>
     <properties>
         <java.version>1.8</java.version>
-        <structured-logging.version>1.9.10</structured-logging.version>
-        <log4j-bom.version>2.16.0</log4j-bom.version>
+        <structured-logging.version>1.9.11</structured-logging.version>
```

Comment:
Can now be updated to the latest version 1.9.12

Reply:
Yeah just realised this sorry.  Pushed changes now!!!